core/next.config.js

```diff
@@ -28,6 +28,17 @@ const nextConfig = {
       });
     }
 
+    // Limit the number of chunks to reduce CDN requests, which contribute to Edge Request costs
+    if (!isServer) {
+      config.plugins.push(
+        new (require('webpack').optimize.LimitChunkCountPlugin)({
+          maxChunks: process.env.WEBPACK_MAX_CHUNKS
```

Comment:
If this requires a new build, do we even need a environment variable for this? I'd be more inclined to just inline a value.